deploy/services-demo/demo.sh

```diff
@@ -95,6 +112,20 @@ function run_nginz() {
         | sed -e "s/^/$(tput setaf ${colour})[nginz] /" -e "s/$/$(tput sgr0)/" &
 }
 
+function copy_brig_templates() {
+    # Need to copy over the templates from Brig since symlinking does not
+    # work with Docker
+    mkdir -p "${SCRIPT_DIR}/resources/templates"
+    cp -r "${SCRIPT_DIR}/../../services/brig/deb/opt/brig/templates/"* "${SCRIPT_DIR}/resources/templates/"
+}
+
+function copy_nginz_configs() {
+    # Need to copy over the configs from Nginz since symlinking does not
+    # work with Docker
```

Comment:
> got the same errors from nginx in the console:

`wire-server/deploy/services-demo/conf/nginz/zwagger-ui` is a symlink to `../../../../services/nginz/zwagger-ui/`, which is a directory under nginz populated by `cd services/nginz && make zwagger-ui/swagger-ui`. I probably had this populated as I've built nginz locally before, and it's not part of the nginz docker target. You could use something like this here:

```suggestion
    # work with Docker
    # ensure swagger UI files are downloaded befory copying
    make -C "${TOP_LEVEL}/services/nginz zwagger-ui/swagger-ui"
```